Library/Formula/ruby-build.rb

```diff
@@ -2,7 +2,7 @@
 
 class RubyBuild < Formula
   homepage "https://github.com/sstephenson/ruby-build"
-  url "https://github.com/sstephenson/ruby-build/archive/v20150506.tar.gz"
+  url "https://github.com/sstephenson/ruby-build/archive/v20150519.tar.gz"
   sha256 "68bc8577897f0a7f76b02a7a666d613de9d15379f5b8b7c6f753c75bd9fade22"
```

Comment:
Updated. Sorry, my release script wasn't aware that homebrew switched to `sha256` in the meantime.